docs/src/starknet/account.md

```diff
@@ -148,7 +116,36 @@ legacy: false
 [...]
 ```
 
-### Custom Account Contract
+### [`account delete`](../appendix/sncast/account/delete.md)
+
+Delete an account from `accounts-file` and its associated Scarb profile. If you pass this command, you will be asked to confirm the deletion.
+
+```shell
+$ sncast \
+    --accounts-file my-account-file.json \
+    account delete \
+    --name some-name \
+    --network alpha-sepolia
+```
+
+### [`account add`](../appendix/sncast/account/add.md)
+
+To import an account into the file holding the accounts info (`~/.starknet_accounts/starknet_open_zeppelin_accounts.json` by default), use the `account add` command.
```

Comment:
Maybe this should just be a separate section somewhere on this page similar to this

```md
## Accounts File

By default the file is this ... but you can override it with `--accounts-file` 

maybe a single example there.
```

I think this could maybe be in advanced usecases.

Reply:
Actually we already have that below, so maybe consider extending it and don't add these explanations anywhere else.

Reply:
I think that this paragraph fits well in this place, and creates a full composition of managing account 
```md
## Managing Accounts
### account add
### account list
### account delete
```
I've updated the docs, wdyt?

Reply:
We can keep it, please add an issue to backlog to link to new docs that will be created as part of #2565